.gitignore

```diff
@@ -2,10 +2,16 @@
 
 # Integration Tests build output
 integration-tests/**/fixtures/**/*.tar.gz
-integration-tests/**/fixtures/**/fastly.toml
+integration-tests/**/fixtures/**/pkg/**/fastly.toml
 integration-tests/**/fixtures/**/*.wasm
 integration-tests/**/fixtures/**/*.js
 integration-tests/**/fixtures/**/*.js.map
+integration-tests/**/fixtures/**/compiler_flags
```

Comment:
These files are creating when compiling an integration test manually via make, which is something I've been doing a lot recently as it is a faster feedback loop when developing against a specific test:
```
INIT_JS=extend-from-builtins.ts make -f ../../../../c-dependencies/js-compute-runtime/Makefile initialized-js-compute-runtime.wasm
```